### PR TITLE
Router setup

### PR DESCRIPTION
**Pull request by daveskull81**
# Description

## Type of change

Please delete options that are not relevant.

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Change Status

- [x] Complete, but not tested (may need new tests)

# How Has This Been Tested?

Manual testing of current routing across three components
Also manually tested `PrivateRoute` component works as expected

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by daveskull81** [comment]
[React Router Setup](https://trello.com/c/KpWyFFFg/70-react-router-setup)

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,29 @@
 import React from 'react'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
 import { Container } from '@material-ui/core'
 import { AuthProvider } from './components/auth/Auth'
-// import { Register } from './components'
-// import { Login } from './components'
+import PrivateRoute from './components/auth/PrivateRoute'
+import { Register } from './components'
```
Comment (Memitaru) [inline src/App.js]: Since we're importing these into an index the register and login components can be imported on the same line

Comment (daveskull81) [reply src/App.js]: Ah yes. I always forget that. I can make that change.

Comment (daveskull81) [reply src/App.js]: Made this change.

**Pull request merged**